Formula/sourcery.rb

```diff
@@ -13,11 +13,10 @@ class Sourcery < Formula
   depends_on :xcode => ["8.0", :build]
 
   def install
-    ENV.delete("CC")
-    ENV["SDKROOT"] = MacOS.sdk_path
-    system "swift", "build", "-c", "release"
-    bin.install ".build/release/sourcery"
-    lib.install Dir[".build/release/*.dylib"]
```

Comment:
As I said, it's not an actually GUI app, it's a CLI app wrapped inside *.app for tricking Xcode to bundle all required libSwift*.dylib inside an app bundle

This app can be unwrapped from *.app bundle (Sourcery.app/Contents/MacOS/Sourcery => prefix/bin/sourcery; Sourcery.app/Contents/Frameworks/* => prefix/Frameworks/*) and this will work too, but brew will try to link Frameworks dir to /usr/local/Frameworks/ and this can break other formulas (for example, Sourcery is using sourcekitten as a dependency, so SourceKittenFramework.framework will be built by this formula, but installing `sourcekitten` formula separately will build this formula too)

I didn't found the way how to make brew link only binaries from formula (and keep frameworks inside cellar folder only) so I made it this way.